docs/lsp.md

```diff
@@ -80,7 +84,7 @@ sequenceDiagram
   Note right of LSPIndexer: Copy GlobalState for<br/>the slow path
   LSPLoop --) LSPTypechecker: SorbetWorkspaceEditTask::runSpecial
   LSPTypechecker ->> LSPTypechecker: LSPTypechecker::runSlowPath
-  Note right of LSPTypechecker: Save old GlobalState<br/>and run the pipeline
+  Note right of LSPTypechecker: pipeline::copyForSlowPath<br/>and run the pipeline
```

Comment:
We have one note "pipeline::copyForSlowPath" and another node "Copy GlobalState for the slow path"

not sure whether that's intentional (fine if it is) but if they're actually using the same logic and we wanted to make that clear in the note, now would be a time to do that.

Reply:
Good catch, the `Copy GlobalState...` one is no longer relevant.